tensorflow/contrib/opt/python/training/elastic_average_optimizer.py

```diff
@@ -123,8 +169,9 @@ def __init__(self,
                ea_custom_getter,
                communication_period=10,
                moving_rate=None,
-               rho=None,
+               rho=0.0,
```

Comment:
Why changing the default value here?

Reply:
1. in async mode, rho is not used in original paper
2. in my experiments, rho=0 works well
    when rho=None, default value: self._rho = self._moving_rate / self._opt._learning_rate, may result a large value.  e..g, moving_rate=self.BETA / communication_period / num_worker=0.9/8/4=0.028125, while the initial learning_rate might be 1e-4, which makes rho=281.25, and this makes the workers not able to do perform exploitation.

Reply:
Regardless we shouldn't change default values like this as this affects reproducibility of existing work which uses this code.